R/update-regional.R

```diff
@@ -23,38 +23,43 @@ update_regional <- function(location, excludes, includes, force, max_execution_t
   # Update delays -----------------------------------------------------------
   if (!is.list(location$generation_time)) {
     futile.logger::flog.trace("loading generation_time.rds")
-    location$generation_time <- readRDS(here::here("data", "generation_time.rds"))
+    location$generation_time <- readRDS(here::here("data", "generation_time.rds")) #suggest moving into list def
   }
   if (!is.list(location$incubation_period)) {
     futile.logger::flog.trace("loading incubation_period.rds")
-    location$incubation_period <- readRDS(here::here("data", "incubation_period.rds"))
+    location$incubation_period <- readRDS(here::here("data", "incubation_period.rds")) #suggest moving into list def
   }
   if (!is.list(location$reporting_delay)) {
     if (location$name %in% c("deaths", "regional-deaths")) {
       futile.logger::flog.trace("loading onset_to_death_delay.rds")
-      location$reporting_delay <- readRDS(here::here("data", "onset_to_death_delay.rds"))
+      location$reporting_delay <- readRDS(here::here("data", "onset_to_death_delay.rds")) #suggest moving into list def
     }
     else {
       futile.logger::flog.trace("loading onset_to_admission_delay.rds")
-      location$reporting_delay <- readRDS(here::here("data", "onset_to_admission_delay.rds"))
+      location$reporting_delay <- readRDS(here::here("data", "onset_to_admission_delay.rds")) #suggest moving into list def
     }
   }
 
   # Get cases  ---------------------------------------------------------------
   futile.logger::flog.trace("loading cases")
-  if ("Region" %in% class(location)) {
-    if (is.na(location$covid_regional_data_identifier)) {
-      location$covid_regional_data_identifier <- location$name
+  if (is.na(location$data)) {
```

Comment:
Nice - should switch the "test" dataset to use this instead of grabbing a random dataset and replacing it in the case_modifier lambda.

Reply:
sounds like a good idea.